README.md

```diff
@@ -57,6 +57,7 @@ Examples:
 - [Create a sandbox](./modal-go/examples/sandbox/main.go)
 - [Create a sandbox using a private image from AWS ECR](./modal-go/examples/sandbox-private-image/main.go)
 - [Execute sandbox commands](./modal-go/examples/sandbox-exec/main.go)
+- [Execute sandbox filesystem commands](./modal-go/examples/sandbox-filesystem/main.go)
```

Comment:
This is a minor nit, feel free to address later but I think we should say `Access sandbox file system` or something like that instead of "execute commands" to be more user-focused